docs/testrunner-toolkit/configuration/testcafe.md

```diff
@@ -132,26 +132,135 @@ __Example__:
+```
+
+### `clientScripts`
+__Description__: Injects scripts into all pages visited during the test. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/test-api/fixture/clientscripts.html).
+
+__Type__: *object*
+
+__Example__:
+```yaml
+  clientScripts: ["/path/to/file1", "/path/to/file2"]
+```
+
+### `skipJsErrors`
+__Description__: Ignores JavaScript errors on a webpage. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#skipjserrors).
+
+__Type__: *boolean*
+
+__Example__:
+```yaml
+  skipJsErrors: true
+```
+
+### `quarantineMode`
+__Description__: Enables the quarantine mode for tests that fail. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#quarantinemode).
+
+__Type__: *boolean*
+
+__Example__:
+```yaml
+  quarantineMode: true
+```
+
+### `skipUncaughtErrors`
+__Description__: Ignores uncaught errors and unhandled promise rejections in test code. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#skipUncaughtErrors).
+
+__Type__: *boolean*
+
+__Example__:
+```yaml
+  skipUncaughtErrors: true
+```
+
+### `selectorTimeout`
+__Description__: Specifies the time (in milliseconds) within which selectors attempt to return a node. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#selectorTimeout`).
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  selectorTimeout: 1000
 ```
+
+### `assertionTimeout`
+__Description__: Specifies the time (in milliseconds) TestCafe attempts to successfully execute an assertion if a selector property or a client function was passed as an actual value. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#assertionTimeout).
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  assertionTimeout: 1000
+```
+
+### `pageLoadTimeout`
+__Description__: Specifies the time (in milliseconds) passed after the DOMContentLoaded event, within which TestCafe waits for the window.load event to fire. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#pageLoadTimeout).
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  pageLoadTimeout: 1000
+```
+
+### `stopOnFirstFail`
+__Description__: Stops a test run if a test fails. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#stopOnFirstFail).
+
+__Type__: *boolean*
+
+__Example__:
+```yaml
+  stopOnFirstFail: true
+```
+
+### `disablePageCaching`
+__Description__: Prevents the browser from caching page content. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#disablePageCaching).
+
+__Type__: *boolean*
+
+__Example__:
+```yaml
+  disablePageCaching: true
+```
+
+### `disableScreenshots`
```

Comment:
This is a duplicate? `disableScreenshots` is the first TestCafe config.

Reply:
yep. just removed this one.